### ISSUE TITLE
iPad/Tablet UI Design

### ISSUE THREAD

**Issue by nathancoooper** [jmshrv/finamp#628]
Wasn't sure whether to add this to the UI redesign issue as that's technically closed with the new UI release, but on tablet devices, the UI could do with some tidying up potentially. Below is a screenshot from my iPad.

![image](https://github.com/jmshrv/finamp/assets/100081814/c1dab525-661b-4b4e-abd3-22880de944bd)

**Comment by Chaphasilor** [comment]
The album and artist screens are definitely not finished yet! But larger screens will definitely be tricky.  
Maybe there could be some sort of split-screen going on, with the album info and playback controls on the left, and the list of songs or albums on the right? What do you think?

**Comment by nathancoooper** [comment]
I certainly think it could work well. Below is how Spotify tackles this and I do find that most iPad applications utilise a similar split ratio of menu side panel and larger main content.

![image](https://github.com/jmshrv/finamp/assets/100081814/991aba89-8f99-4fd3-b884-d96ba51ccd4c)

**Comment by jmshrv** [comment]
Yeah a split screen thing would be useful, and would also benefit desktop too :)

**Comment by Chaphasilor** [comment]
I know Spotube has implemented such a sidebar that is adaptive and becomes a nav bar on small screens. Not sure how easy that is to integrate though

**Comment by nathancoooper** [comment]
> Yeah a split screen thing would be useful, and would also benefit desktop too :)

Is there a desktop release? Or is this planned?

**Comment by Chaphasilor** [comment]
@nathancoooper yes, we're working on it. Although it's more of a "hey, this would be neat to have an I'd use it myself", but it won't receive the same level of support as the mobile version does.  
We're still fighting with some playback issues, but otherwise everything's working well enough!

What platform are you on?

**Comment by nathancoooper** [comment]
@Chaphasilor I'm using the iPad Pro 11-inch 2nd Gen. I very much appreciate the support, most applications can't even go full-screen on iPad OS so just having that makes Finamp better than most.

**Comment by phildenhoff** [comment]
_edit: Ah I just realised this is controllable using the "Allow SplitScreen" setting. Sweet! thank you._

I think in #719 you updated the tablet view. IMO, the split view is _fine_ but I strongly prefer having a fullscreened Player.

Having a player / library explorer split view is useful if you want to add music while seeing what's playing (?), but I love the full screen view because I leave my iPad on to show what's playing, and I _only_ want to see that.

As a demo, here's what I see today when I have Finamp full screened

![3919B0B8-DD01-432E-944B-D11896C378D1_4_5005_c](https://github.com/jmshrv/finamp/assets/17505728/dec5d024-07e4-414a-84e6-053b98a8695c)

What I would much rather is this view, which shows up if I shrink the view using Stage Manager

![9939701C-3346-41D1-B8EB-1DA24FAED29E_4_5005_c](https://github.com/jmshrv/finamp/assets/17505728/f265a4aa-7018-4766-a1eb-0fd932808522)

For comparison, this is Plexamp, I think also does this well. First, with the queue closed, and then with it open:

![D7CE8C50-F760-48B2-A38B-E483A8350638_1_102_o](https://github.com/jmshrv/finamp/assets/17505728/2c2bc85c-6a7d-400a-9f2a-6349c88e6623)
![3E3A7AC8-6C8B-4685-B976-8C111FE87150_1_102_o](https://github.com/jmshrv/finamp/assets/17505728/707ae58e-b7b2-4fe5-ac9d-bb3bcab8fbfd)

**Comment by Chaphasilor** [comment]
Glad you found the setting! :)  
Embedding the queue onto the player screen isn't planned yet, but I'm not against it. It's just pretty low on the priority list, hope you understand...
